EndlessSky.xcodeproj/project.pbxproj

```diff
@@ -27,7 +28,6 @@
 		62A405BA1D47DA4D0054F6A0 /* FogShader.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 62A405B81D47DA4D0054F6A0 /* FogShader.cpp */; };
 		62C3111A1CE172D000409D91 /* Flotsam.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 62C311181CE172D000409D91 /* Flotsam.cpp */; };
 		6A5716331E25BE6F00585EB2 /* CollisionSet.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 6A5716311E25BE6F00585EB2 /* CollisionSet.cpp */; };
-		938E07E4235970CE00DC2C2B /* FormationPattern.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 938E07E0235970CE00DC2C2B /* FormationPattern.cpp */; };
```

Comment:
this change seems unrelated?

Reply:
The xcode patch I assume Derpy has used sometimes does that, I think.

Reply:
ok

Reply:
@RisingLeaf check this one out https://github.com/endless-sky/endless-sky/pull/7612/files#diff-f99692e65fcaf0033fc339558d87b2a512a27f2396e27bb6932887744392d967

Reply:
Uh, you know, I am working with xcode amd it does that automatical, yeah I get it, i should not have complained